app/lib/smaxtec_api.rb

```diff
@@ -4,7 +4,7 @@ class SmaxtecApi
 
   SMAXTEC_API_EMAIL = Rails.application.secrets.smaxtec_api_email
   SMAXTEC_API_PASSWORD = Rails.application.secrets.smaxtec_api_password
-  SMAXTEC_API_BASE_URL = 'https://api-staging.smaxtec.com/api/v1'
```

Comment:
The different url won't match the VCR cassette and because of that, the tests are failing. When you have resolved the issue, you can merge